### ISSUE TITLE
Question about Hyper-Parameters

### ISSUE THREAD

**Issue by guanhuankang** [XinyuYanTJU/Ranking-Video-Salient-Object-Detection#2]
I am writing to ask for the training hyper-parameters, such as **batch size, learning rate, optimizer, number of epochs**.  Although I found some of these settings in your code, it seems like they are not the one your used (since I found the nb_train in your code is set to 321, epochs=20, number of frames per batch = 3 and learning rate=1e-5, which means that your model only see 321*20*3=19260 images under learning rate 1e-5. Besides, I use this setting to retrain the model, it ran really fast (less than 10mins, which is much faster than your reported training time 30hours), but it doesn't converge! ). Hence, I am writing to ask for the hyper-parameters you used to get the paper's results. 

Thanks!
